solidity/contracts/KeepBonding.sol

```diff
@@ -109,29 +121,43 @@ contract KeepBonding {
     }
 
     /// @notice Withdraws amount from operator's value available for bonding.
-    /// Can be called only by the operator or by the stake owner.
+    /// Can be called only by:
+    /// - operator
+    /// - staked tokens owner
```

Comment:
It's not possible in case of a token grant. Can we update docs to be explicit about cases that are supported?

Reply:
Please let me know what do you think about this:
```
Withdraws amount from operator's value available for bonding.
Should not be used by grantee of managed grants. For this case,
please use `withdrawAsManagedGrantee` instead.

This function can be called only by: 
- operator,
- liquid, staked tokens owner (not a grant),
- direct staked tokens grantee (not a managed grant).
```

Reply:
Looks fine! Thanks.